.github/workflows/docker-release-image-publish.yml

```diff
@@ -10,6 +10,7 @@ on:
 jobs:
     build-push:
         name: Build & push Docker release image
+        if: github.repository == 'PostHog/posthog'
```

Comment:
With this one, it was missing the `if` guard for the whole workflow like some of the others, so I've added it to match, though let me know if this was in error and I can revert.